app/code/Stanislavz/AskQuestion/Helper/Data.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Stanislavz\AskQuestion\Helper;
+
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Store\Model\ScopeInterface;
+
+/**
+ * Class Data
+ * @package Stanislavz\AskQuestion\Helper
+ */
+class Data extends AbstractHelper
+{
+    public const XML_PATH_STANISLAVZ_CRON_ENABLE = 'stanislavz_crone_options/cron/enable';
```

Comment:
Please rename this section  'stanislavz_crone_options/cron/enable'  >  'stanislavz_cron_options/cron/enable';